supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendSingletonActor.scala

```diff
@@ -1,12 +1,12 @@
 package cromwell.backend.impl.jes
 
 import akka.actor.{Actor, ActorLogging, Props}
-import cromwell.backend.impl.jes.statuspolling.{JesApiQueryManager}
+import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager
 import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.DoPoll
 
-class JesBackendSingletonActor extends Actor with ActorLogging {
+final case class JesBackendSingletonActor(jesConfiguration: JesConfiguration) extends Actor with ActorLogging {
```

Comment:
But case classes & adts aren't isomorphic. The idiomatic way to implement an adt in scala is via case classes but one can use case classes for other things.

I've long felt that case classes should have been the default class, and class the opt-in. The reason being that they provide a set of reasonable default behavior that one should generally want. public vals for the items in the constructor, a default apply(), inability to extend, a usually not completely stupid equals/toString, etc.